v1_20_R4/src/main/java/de/corneliusmay/silkspawners/nms/v1_20_R4/NMSHandler.java

```diff
@@ -0,0 +1,41 @@
+package de.corneliusmay.silkspawners.nms.v1_20_R4;
+
+import de.corneliusmay.silkspawners.api.NMS;
+import org.bukkit.Material;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.attribute.AttributeInstance;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.RayTraceResult;
+
+public class NMSHandler implements NMS {
+
+    @Override
+    public Block getTargetBlock(Player player) {
```

Comment:
Could you please explain the reasoning behind replacing return player.getTargetBlockExact(5); with the new code block? Specifically, I am not clear on the use of RayTraceResult and its benefits over the original method. A detailed explanation would be appreciated.